converter/internal/promtailconvert/promtailconvert.go

```diff
@@ -132,20 +132,11 @@ func appendScrapeConfig(
 ) {
 
 	b := build.NewScrapeConfigBuilder(f, diags, cfg, gctx)
-	b.Validate()
 	b.Sanitize()
 
 	// Append all the SD components
-	b.AppendKubernetesSDs()
-	b.AppendDockerSDs()
 	b.AppendStaticSDs()
```

Comment:
static SD configs didn't 'just work' and requires a little more digging to wire up. aiming for a future PR if we decide to do it.

Reply:
Let's add a TODO comment to remember why we're leaving this here, WDYT?

Reply:
I have figured out what was not working and implemented this in my last commit.